core/frontend/src/main/scala/io/udash/bindings/inputs/CheckButtons.scala

```diff
@@ -0,0 +1,99 @@
+package io.udash.bindings.inputs
+
+import io.udash._
+import io.udash.properties.PropertyCreator
+import io.udash.properties.seq.SeqProperty
+import org.scalajs.dom.{Element, Event, Node}
+import org.scalajs.dom.html.{Div, Input => JSInput}
+import scalatags.JsDom.TypedTag
+import scalatags.JsDom.all._
+
+/** Checkboxes for finite options with many elements selection. Bound to SeqProperty. */
+object CheckButtons {
+  def inputsOnlyDecorator[T]: Seq[(JSInput, T)] => Seq[Node] = RadioButtons.inputsOnlyDecorator
+  def spanWithLabelDecorator[T](labelFactory: T => Modifier): Seq[(JSInput, T)] => Seq[Node] =
+    RadioButtons.spanWithLabelDecorator(labelFactory)
+  def divWithLabelDecorator[T](labelFactory: T => Modifier): Seq[(JSInput, T)] => Seq[Node] =
+    RadioButtons.divWithLabelDecorator(labelFactory)
+
+  /**
+    * @param selectedItems SeqProperty which gonna be bound to checkboxes
+    * @param options Seq of available options, one checkbox will be created for each option.
+    * @param decorator Function creating HTML element from checkboxes Seq.
+    * @param xs Modifiers to apply on each generated checkbox.
+    * @return HTML element created by decorator.
+    */
+  def apply[T : PropertyCreator](
+    selectedItems: SeqProperty[T, _ <: ReadableProperty[T]], options: ReadableProperty[Seq[T]]
+  )(decorator: Seq[(JSInput, T)] => Seq[Node], xs: Modifier*): InputBinding[Div] = {
```

Comment:
Modifiers usually work nicely as a separate list. Also, there are a few better names than `xs` :rofl:

Reply:
I want too keep all these applies similar. First list describes behavior, second apperance of element.